### DISCUSSION TITLE
Clarification on Extract Thinker Framework and Supported LLM Models

### DISCUSSION THREAD

**Discussion by harinisri2001** [enoch3712/ExtractThinker#154] [General]
Hi @enoch3712 , I'm exploring the Extract Thinker framework and would like to confirm whether my understanding is correct. I also have a few queries.

My understanding is that we use tools like Tesseract or Docling to extract content from documents, and then we call LLMs to process and provide structured information. Is this accurate?

Additionally, could you clarify which LLM models are supported in the framework? What additionally things can be achieved with Extarct Thinker?

**Comment by enoch3712** [comment]
Hello @harinisri2001! 

>  My understanding is that we use tools like Tesseract or Docling to extract content from documents, and then we call LLMs to process and provide structured information. Is this accurate?

Yes that's correct! Thats the core functionality. 

> Additionally, could you clarify which LLM models are supported in the framework? What additionally things can be achieved with Extarct Thinker?

+100. Is build on top of Litellm, PydanticAI soon 


ExtractThinker is IDP (Intelligent Document Processing) for LLMs. Similar to Langchain, that is too generalized, but for just for the narrow use case of IDP, with all the features required. 

RAG ofc is not included and never will :)

**Comment by harinisri2001** [comment]
@enoch3712 Thank you so much for your clarification.
